README.md

```diff
@@ -48,3 +48,13 @@ OR
 
 Perform: `sm2 --start MERCHANDISE_IN_BAGGAGE_ALL` to run all mods services, and then separately run the UI tests.
 (won't test the branch)
+
+## Accessibility Tests
+
+### Prerequisites
+Have node installed on your machine
+
+### Execute tests
+To run the tests locally, simply run:
+```bash
+sbt 
```

Comment:
missing command itself 
And also missing License

Reply:
done